orchagent/pfcwdorch.cpp

```diff
@@ -657,16 +667,14 @@ PfcWdSwOrch<DropHandler, ForwardHandler>::PfcWdSwOrch(
 {
     SWSS_LOG_ENTER();
 
-    string platform = getenv("platform") ? getenv("platform") : "";
-    if (platform == "")
-    {
-        SWSS_LOG_ERROR("Platform environment variable is not defined");
-        return;
-    }
-
     string detectSha, restoreSha;
-    string detectPluginName = "pfc_detect_" + platform + ".lua";
-    string restorePluginName = "pfc_restore.lua";
+    string detectPluginName = "pfc_detect_" + this->m_platform + ".lua";
+    string restorePluginName;
+    if (this->m_platform == CISCO_8000_PLATFORM_SUBSTRING) {
```

Comment:
can this cover both fixed and chassis sub-system ?

Reply:
Yes for now. If there is any change in future, we will update it here.